### ISSUE TITLE
crack-volume3.sh test hangs

### ISSUE THREAD

**Issue by itepastra** [glv2/bruteforce-luks#41]
The test does not seem to check any passwords if `-t` is 3 or more. This makes it hang indefinitely, which is not desired behaviour. When `-t 1` or `-t 2` is used in the test instead everything works as expected

**Comment by pasom-creator** [comment]
Correct, I am cracking my Luks2 partition right now and have the same issue. If threads more then 2, app hangs.
